spec/datadog/appsec/configuration/settings_spec.rb

```diff
@@ -77,6 +77,30 @@ def patcher
       end
     end
 
+    describe '#rasp_enabled' do
+      subject(:rasp_enabled) { settings.appsec.rasp_enabled }
```

Comment:
I think it's obsolete now

Reply:
thanks, removed it [7eeab83](https://github.com/DataDog/dd-trace-rb/pull/4311/commits/7eeab83d9861ec7d955c72cc3a598221c11aa1b5)